c2corg_ui/static/js/map/map.js

```diff
@@ -59,7 +59,14 @@ app.module.directive('appMap', app.mapDirective);
  * @export
  * @ngInject
  */
-app.MapController = function($scope, mapFeatureCollection) {
+app.MapController = function($scope, mapFeatureCollection, $attrs) {
+
+
+  /**
+   * @type {number}
+   * @private
+   */
+  this.zoom_ = $attrs['zoom'];
```

Comment:
I proposed a map because I put there an easter egg (waypoint lost in the bermuda's triangle)  ;) 
I needed this change while I was working on this PR.